### DISCUSSION TITLE
winget installs x86 version instead of x64

### DISCUSSION THREAD

**Discussion by Silther** [Rem0o/FanControl.Releases#2437] [General]
is it normal that the winget installer uses the x86 version instead of the x64 version>?

**Comment by Rem0o** [comment]
FanControl is compiled as "AnyCPU", there is no seperate x86 or x64 version.

**Comment by Silther** [comment]
So it's probably detected as x86 by BCUninstaller because it was installed in the "Programm files (x86)" folder
